Autonomous_Parallel_Parking/Autonomous_Parallel_Parking.c

```diff
@@ -15,14 +15,84 @@
  ****************************************************************************************************************/
 
 #include "cpu.h"
-#include "board_led.h"
+#include "pin_setup.h"
 
-#include "foo.h"
+int count = 0;
 
 int main()
 {
   cpu_init();
-  board_led_init();
-
+  GPIOInterruptSetup();
   return 0;
 }
+
+/**
+* @brief IRQ for PA0 (user button) - to test if PA0 interrupt is functioning.  Turns on blue LED every 5 times the 
+* button is pressed.
+*/
+void EXTI0_IRQHandler(void){
```

Comment:
Yea it is for testing but I kept it there in case anyone wants to modify it to test anything that they're working on.  And you mean make a separate header/C file for the gpio interrupt setup function and another one for the IRQ's?